docs/customize/Tokenizers.md

```diff
@@ -106,6 +111,17 @@ The following is a list of preprocessors that are shipped with Nominatim.
         heading_level: 6
         docstring_section_style: spacy
 
+::: nominatim_api.query_preprocessing.regex_replace
+    options:
+        members: False
+        heading_level: 6
+        docstring_section_style: spacy
+    description: 
+        This option runs any given regex pattern on the input and replaces values accordingly
+    replacements:
+        - pattern: regex pattern
+          replace: string to replace with
+
```

Comment:
Have you compiled the documentation and looked at the result of your changes?

Reply:
The documentation is built using mkdocs, see the [Developer's documentation](https://nominatim.org/release-docs/develop/develop/Development-Environment/#documentation-pages). This page also has a list of packages you need to install to make it work.